frontend/src/views/ShootList.vue

```diff
@@ -205,7 +205,7 @@ export default {
       const key = value
       await this.setShootListFilter({ filter: key, value: !this.getShootListFilters[key] })
 
-      this.$localStorage.setObject('project/_all/shoot-list/filter', pick(this.getShootListFilters, ['onlyShootsWithIssues', 'progressing', 'userIssues', 'deactivatedReconciliation', 'hideTicketsWithLabel']))
+      this.$localStorage.setObject('project/_all/shoot-list/filter', pick(this.getShootListFilters, ['onlyShootsWithIssues', 'progressing', 'noOperatorAction', 'deactivatedReconciliation', 'hideTicketsWithLabel']))
```

Comment:
```suggestion
      this.$localStorage.setObject('project/_all/shoot-list/filter', pick(this.getShootListFilters, [
        'onlyShootsWithIssues',
        'progressing',
        'noOperatorAction',
        'deactivatedReconciliation',
        'hideTicketsWithLabel'
      ]))
```
or in addition, extract to variable to make it more readable